project/assets/main/creatures/primary/richie/pomodoro.json

```diff
@@ -48,7 +48,7 @@
     },
     {
       "who": "#sensei#",
-      "text": "Oh, lighten up #player#./ Sounds like someone needs a Pomodoro."
+      "text": "Oh, lighten up #player#. Sounds like someone needs a Pomodoro."
```

Comment:
Do replaceable entities like '#' get fouled up by lull characters now?

Reply:
No; the lull character logic only interacts with specific punctuation like '!', '.', '?', ',' and '-'. It does not interact with hash symbols